authentication/authentication-service/src/main/resources/bootstrap.properties

```diff
@@ -1,7 +1,7 @@
 
 # Application name - the name appended at starting of file name to differentiate
 # between different property files for different microservices
-spring.application.name=id-authentication

```

Comment:
can we have the name as `id-authentication-external` ?

Reply:
done